model/supervised_finetuning/configs/config.yaml

```diff
@@ -18,7 +18,19 @@ defaults:
   datasets:
     - webgpt
     - prompt_dialogue
-  cache_dir: ~/.cache
+    - squad_v2
+    - adversarial_qa
+    - trivia_qa_nocontext
+    - xsum
+    - cnn_dailymail
+    - prompt_dialogue
+    - multi_news
+    - scitldr
+    - soda
+    - joke
+    -
+    - joke
```

Comment:
typo?

Reply:
Ah, yes it is, I accidentally added the config. Guess I will just update it to use my current datasets. These have been running for hours without issue on codegen 2B